src/sources/soundsourceffmpeg.cpp

```diff
@@ -474,11 +559,34 @@ bool SoundSourceFFmpeg::readFramesToCache(unsigned int count, SINT offset) {
                     m_SStoredJumpPoint = nullptr;
                 }
 
+#if LIBAVFORMAT_CHANGE_AVSTREAM
+                // AVERROR(EAGAIN) means that we need to feed more
+                // That we can decode Frame or Packet
+                do {
```

Comment:
Infinite loop (FFmpeg 3.1.5): Nested do/while loops seem to get stuck during CoverArtUtilTest for .m4a